packages/core/ios/Sources/DdSdk.swift

```diff
@@ -152,6 +152,13 @@ class RNDdSdk: NSObject {
             _ = ddConfigBuilder.set(sampleTelemetry: telemetrySampleRate)
         }
 
+        if let threshold = configuration.nativeLongTaskThresholdMs as? TimeInterval {
+            if (threshold != 0) {
```

Comment:
I think it would be better to pass this logic to the native SDK side in order to match the Android implementation and not have it done in RN side.

Reply:
I agree. 
As this will require a new iOS SDK release however, I would do that in a separate PR so we can release js long task monitoring earlier. Is it ok for you?